diff --git a/CMakeLists.txt b/CMakeLists.txt
index eca17957..f576d62a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -368,7 +368,7 @@ else()
   if (USE_PLATFORM_UUID)
     if (APPLE)
       target_compile_definitions(${PROJECT_NAME} PRIVATE GUID_CFUUID HAVE_UNISTD_H=1)
-      target_link_libraries(${PROJECT_NAME} "-framework Foundation")
+      target_link_libraries(${PROJECT_NAME} PRIVATE "-framework Foundation")
     else()
       target_compile_definitions(${PROJECT_NAME} PRIVATE GUID_LIBUUID)
       target_link_libraries(${PROJECT_NAME} PRIVATE uuid)
